pkgs/applications/audio/ams/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchgit
+, automake
+, alsaLib
+, ladspaH
+, libjack2
+, fftw
+, zita-alsa-pcmi
+, qt4
+, pkg-config
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "ams";
+  version = "2.1.2";
+
+  src = fetchgit {
+    url = https://git.code.sf.net/p/alsamodular/ams.git;
+    sha256 = "0v5lasrgghcjw47dfxzpxp93mxykimh3raz8kpzk2i7z7cnlp5wa";
+    rev = "a47d45272a1c387c1cfc37b9e82d6245429a3117";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook ];
+
+  buildInputs = [ alsaLib ladspaH libjack2 fftw zita-alsa-pcmi qt4 ];
+
+  # From Debian package
+  # TODO fetch properly from source, if possible
+  patches = [ ./Make-vocoder-module-compatible-to-C-11.patch ];
```

Comment:
Looks like this is merged to master along with other fixes: https://sourceforge.net/p/alsamodular/ams.git/ci/master/log/?path=

I would just suggest using https://sourceforge.net/p/alsamodular/ams.git/ci/3250bbcfea331c4fcb9845305eebded80054973d/ for `rev` and changing version accordingly: https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
e.g. `ams-unstable-2019-04-27`

Reply:
Actually, the upstream repo is named `alsamodular-ams`, so `alsamodular-ams-unstable-2019-04-27` ?

Reply:
`ams-unstable-2019-04-27` is sufficient IMO.

Reply:
Unless there are multiple projects named ams.